ldap/config.go

```diff
@@ -29,6 +29,14 @@ const (
 
 	// DefaultTLSMaxVersion for the ClientConfig.TLSMaxVersion
 	DefaultTLSMaxVersion = "tls12"
+
+	// DefaultOpenLDAPUserPasswordAttribute defines the attribute name for the
+	// openLDAP default password attribute which will always be excluded f
```

Comment:
Additional "f" here:

```suggestion
	// openLDAP default password attribute which will always be excluded
```

Reply:
ty!